python/tests/stress/arcticdb/version_store/test_stress_multicolumn.py

```diff
@@ -38,7 +38,16 @@ def make_periods(start_date, end_date, freq, range_type="b"):
     return [r for r in ranges if len(r) > 0]
 
 
-@pytest.mark.parametrize("lib_type", ["lmdb_version_store_big_map", "s3_version_store", "s3_version_store"])
+@pytest.mark.parametrize(
+    "lib_type",
+    [
+        "lmdb_version_store_big_map",
+        "s3_version_store",
+        "s3_version_store",
+        "azure_version_store",
+        "azure_version_store",
```

Comment:
Have you figured out why `s3_version_store` was duplicated to begin with?

Reply:
I have figured this is a bad idea to just blindly copying everything from S3. So I have removed it in later commit.